salt/modules/aptpkg.py

```diff
@@ -221,10 +240,7 @@ def latest_version(*names, **kwargs):
         cmd = ['apt-cache', '-q', 'policy', name]
         if repo is not None:
             cmd.extend(repo)
-        out = __salt__['cmd.run_all'](cmd,
-                                      output_loglevel='trace',
-                                      python_shell=False,
-                                      env={'LC_ALL': 'C', 'LANG': 'C'})
+        out = _call_apt(cmd, scope=False)
```

Comment:
This will force a user to set `LC_ALL` and `LANG`. Would it be a good idea to set up some sort of default environment? Since we're passing `globals()` to `get_module_environment()`, we could maybe define it as a dunder in the global scope?

Reply:
These are just a left-overs from older times, since we're now got `run_all` properly and resetting the locale there right away.